app/vue/package.json

```diff
@@ -70,17 +70,22 @@
     "url-loader": "^0.6.2",
     "util-deprecate": "^1.0.2",
     "uuid": "^3.1.0",
-    "vue": "^2.4.2",
     "vue-hot-reload-api": "^2.1.1",
-    "vue-loader": "^12.2.1",
     "vue-style-loader": "^3.0.1",
-    "vue-template-compiler": "^2.4.2",
     "webpack": "^3.6.0",
     "webpack-dev-middleware": "^1.12.0",
     "webpack-hot-middleware": "^2.19.1"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
-    "nodemon": "^1.12.1"
+    "nodemon": "^1.12.1",
+    "vue": "^2.5.2",
+    "vue-loader": "^13.3.0",
+    "vue-template-compiler": "^2.5.2",
+  },
+  "peerDependencies": {
+    "vue": "*",
```

Comment:
Can we check that we work with **any** vue version?

Reply:
I think that we should be checked vue latest version only. because `vue-template-compiler` forces the same version as `vue`

Reply:
I mean, do we work with vue version 1? If not, we shouldn't put an asterisk here